src/cls/IPM/Utils/FileBinaryTar.cls

```diff
@@ -321,13 +321,20 @@ ClassMethod ConstructTar(Path As %String, RelativeTo As %String, archive As %Str
     set flag = 5 // Directory
     set size = 0
     set name = ##class(%File).GetFilename($Extract(Path,1,*-1))
+    set prefix = $Extract(prefix,1,*-$Length(name)-1)
+    if $$$isWINDOWS {
+      set prefix = $Replace(prefix,"\","/")
```

Comment:
Is this backwards?

Reply:
This is intentional - we need to have unix-style slashes in the archive, so on Windows we replace Windows slashes with Unix slashes.

Reply:
@isc-tleavitt  Please add a comment explaining this in the code since it is counter-intuitive.